README.md

```diff
@@ -80,92 +90,82 @@ Alternatively, you can install it directly from git:
 cargo install --git https://github.com/nimiq/core-rs-albatross.git
 ```
 
-## Documentation
-Extensive documentation explaining how the protocol is built up, the JSON-RPC specification, how to get started building applications on top of Nimiq and more can be found at the [Nimiq Developer Center](https://www.nimiq.com/developers/).
-
-## Software Integrity and Authenticity
-You can refer to the [documentation](./build/README.md) to learn more about reproducing Nimiq software for yourself as well as checking software signatures in order to verify the integrity and authenticity of software.
-
-## Usage
-
-After installation, you can run the client directly, like this:
+### Configuration
+You need a configuration file to customize your node according to your specific requirements. Follow one of the methods below to create and edit your configuration file.
 
+**Option A**
+The configuration file is generated automatically and in a specific location.
+1. Generate the configuration file with the following command:
```

Comment:
This is not correct. If you run  `cargo run --release --bin nimiq-client` without supplying the config file it will fail.
So I propose to just use the sample configuration file as in 2 and copy it/ modify it accordingly